pages/documentation/migration-V3.md

```diff
@@ -483,68 +480,64 @@ We improved the accessibility of the footer by converting a non-semantic heading
+   Each of the files below is the most current version of the USWDS default settings. For each of these files, use the diff tool to find the differences between your settings file and the current defaults.
 
-1. Once you have all your project-specific settings in a single file, we'll load these customizations into the USWDS core engine. We do this with a special `@use` statement:
+   - [Color settings](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-color.scss)
+   - [Component settings](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-components.scss)
+   - [General settings](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-general.scss)
+   - [Spacing settings](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-spacing.scss)
+   - [Typography settings](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-typography.scss)
+   - [Utilities settings](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-utilities.scss)
+  
+   Look for instances where your project has a different value than the default. Ignore cases where a setting exists in the current version but not in your version. This indicates a new setting that probably does not apply to your project. Instances where a setting exists in your version but not in the current version can likely be ignored. This usually indicates a deprecated setting that should not affect your project, but it can be worth checking to see if that variable appears anywhere else in your codebase!
+
+   When you see a setting that appears different from the current default, this is probably one of your project's custom settings. Copy this setting and add it to your new `_uswds-theme.scss` file.
+
+   At the end of this process, your new `_uswds-theme.scss` file will look something like the following:
+
+   ```scss
+   $theme-image-path: "../uswds/img";
+   $theme-font-path: "../uswds/fonts";
+   $theme-show-compile-warnings: false;
+   $theme-show-notifications: false;
+   $theme-focus-color: "blue-50v";
+   $theme-global-paragraph-styles: true;
+   $theme-global-link-styles: true;
+   $theme-global-content-styles: true;
+   $theme-utility-breakpoints: (
+    "card": false,
+    "card-lg": true,
+    "mobile": true,
+    "mobile-lg": true,
+    "tablet": true,
+    "tablet-lg": true,
+    "desktop": true,
+    "desktop-lg": true,
+    "widescreen": true
+   ); 
+   ```
+4. **Load these customizations into USWDS core.** Once you have all your project-specific settings in a single file, we'll load these customizations into the USWDS core package. We do this with a special `@use ... with` statement:
   
-    {:.site-terminal}
```

Comment:
Removed the `.site-terminal` styles to create a consistent presentation that matches the code diff styling.